drivers/wifi/winc1500/wifi_winc1500_nm_bus_wrapper.c

```diff
@@ -87,6 +94,37 @@ static s8_t spi_rw(u8_t *mosi, u8_t *miso, u16_t size)
 
 #endif
 
+struct winc1500_gpio_configuration *winc1500_configure_gpios(void)
+{
+	const int flags_int_in = (GPIO_INPUT |
+				  DT_INST_0_ATMEL_WINC1500_IRQ_GPIOS_FLAGS);
+	const int flags_noint_out = GPIO_OUTPUT_ACTIVE;
+	struct device *gpio_en, *gpio_irq, *gpio_reset;
+
+	gpio_en = device_get_binding(
+		DT_INST_0_ATMEL_WINC1500_ENABLE_GPIOS_CONTROLLER);
+	gpio_irq = device_get_binding(
+		DT_INST_0_ATMEL_WINC1500_IRQ_GPIOS_CONTROLLER);
+	gpio_reset = device_get_binding(
+		DT_INST_0_ATMEL_WINC1500_RESET_GPIOS_CONTROLLER);
+
+	gpio_pin_configure(gpio_en,
+			   winc1500_gpios[WINC1500_GPIO_IDX_CHIP_EN].pin,
+			   flags_noint_out);
```

Comment:
Use `GPIO_OUTPUT_ACTIVE | DT_INST_0_ATMEL_WINC1500_ENABLE_GPIOS_FLAGS` i.e. include the devicetree flags in the configuration.  It's the devicetree flags that tell things what the active level is, and we can expect them to be set correctly.

Also for RESET, though that should perhaps be `GPIO_OUTPUT_INACTIVE`.  It doesn't seem to make sense to initialize the chip to enabled and reset.

Reply:
actually, it should be GPIO_OUTPUT for both